manifests/configmap.yaml

```diff
@@ -16,6 +16,7 @@ data:
   super_username: postgres
   enable_teams_api: "false"
   spilo_privileged: "false"
+  binaries_path_template: "/usr/lib/postgresql/%s/bin"
```

Comment:
I see that you've added also a new field to the operator configuration, but I'm afraid it's not used, since it's not in the `Config` data structure. It's totally make sense to have it on the operator level (in case if one wants to change the binary path for all the databases), but it needs to be present in `Config`, corresponding CRD for configuration and Helm chart (these are different ways of configuring the operator) taking into account that it can be overwritten on the database config level. Could you add it too?

Reply:
Just fyi, [here](https://github.com/zalando/postgres-operator/blob/master/docs/developer.md#introduce-additional-configuration-parameters) is a list what has to be done when introducing new config parameters.